### DISCUSSION TITLE
How to properly define Action custome view

### DISCUSSION THREAD

**Discussion by lepresk** [filamentphp/filament#7400] [Help]
How to properly define a custom view on an Action ?

I tried with `Action::make('action')->view('componets.custom')` my view componets.custom.blade.php is displayed but when clicked nothing happens, the action does not react anymore.

**Comment by danharrin** [comment]
https://filamentphp.com/docs/3.x/actions/modals#custom-modal-content

**Comment by lepresk** [comment]
I would like to change the appearance of the button to feet my need, not the modal content.

![Capture](https://github.com/filamentphp/filament/assets/10780686/46baba5a-9441-47b7-af23-bfb5738cbc36)

**Reply by danharrin** [reply]
So what did you put in the view

**Comment by leandrocfe** [comment]
maybe you can use `->link()` and customize the button using `->extraAttributes()`
```php
->link()
->extraAttributes(['class' => 'bg-primary-500 rounded-lg text-white py-2 ...'])

```

**Reply by lepresk** [reply]
Thank, I'll deal with it
